RollbarNotifier/Sources/RollbarReport/Report/BinaryImage.swift

```diff
@@ -59,6 +59,9 @@ struct BinaryImage: RawRepresentable {
             .compactMap { rawValue[any: $0] }
 
         let addr = Address.binary(rawValue[any: "image_addr", default: 0])
-        self.addr = (addr, addr + self.size - 1)
+        guard let end = addr + (self.size - 1) else {
```

Comment:
The parenthesis is unnecessary equationally, moreover if for some reason self.size (which is unsigned) where to be 0, the - 1 would cause an overflow, right?